### ISSUE TITLE
Skip parameter ignored

### ISSUE THREAD

**Issue by antonGritsenko** [Azure/arm-ttk#33]
Repro:

```
Test-AzTemplate -TemplatePath $TemplateFileOrFolder -Skip "IDs-Should-Be-Derived-From-ResourceIDs"
```

Result: test still performed (there are no any references to $skip in the code)

**Comment by bmoore-msft** [comment]
@StartAutomating - did this fix get checked in yet?

**Comment by StartAutomating** [comment]
@bmoore-msft - just created a PR - https://github.com/Azure/arm-ttk/pull/44

**Comment by bmoore-msft** [comment]
merged #44
